JsonSubTypes/JsonSubtypesConverter.cs

```diff
@@ -47,7 +47,19 @@ internal class JsonSubtypesConverter : JsonSubtypes
             _addDiscriminatorFirst = addDiscriminatorFirst;
             foreach (var type in _subTypeMapping)
             {
-                _supportedTypes.Add(type.Value, type.Key);
+                if (_supportedTypes.ContainsKey(type.Value))
+                {
+                    if (_addDiscriminatorFirst)
```

Comment:
`_addDiscriminatorFirst` or `_serializeDiscriminatorProperty` ?

Reply:
Yes, it should be _serializeDiscriminatorProperty